spec/dummy/client/app/components/HelloWorld.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 // Super simple example of the simplest possible React component
-export default class HelloWorld extends React.Component {
```

Comment:
Why do we need to split definition and export if we don't do anything with exported class?